fhir-search/src/main/java/com/ibm/fhir/search/util/ReferenceUtil.java

```diff
@@ -30,8 +30,8 @@
     private static final String URN = "urn:";
 
     // A set of resource type names to assist computing the base URL string
-    private static final Set<String> resourceTypes = Arrays.stream(FHIRResourceType.ValueSet.values())
-            .map(FHIRResourceType.ValueSet::value)
+    private static final Set<String> resourceTypes = Arrays.stream(FHIRResourceType.Value.values())
+            .map(FHIRResourceType.Value::value)
```

Comment:
I think there was an early piece of code that did a similar thing and used ModelSupport with the false.  Do we want that here too?

Reply:
sure, i can make that change

Reply:
done